mgmt/rest/v1/tribe.go

```diff
@@ -42,12 +42,16 @@ var (
 	ErrMemberNotFound        = errors.New("Member not found")
 )
 
+// Deprecated: Update to apiV2(https://github.com/intelsdi-x/snap-client-go).
```

Comment:
There is no Tribe routes in API V2, but before removing API V1, Tribe should be supported in API V2 too.